Gemfile

```diff
@@ -28,11 +28,8 @@ group :development do
   gem "puppet-module-win-default-r#{minor_version}", '~> 0.3',   require: false, platforms: [:mswin, :mingw, :x64_mingw]
   gem "puppet-module-win-dev-r#{minor_version}", '~> 0.3',       require: false, platforms: [:mswin, :mingw, :x64_mingw]
   gem "github_changelog_generator",                              require: false, git: 'https://github.com/skywinder/github-changelog-generator', ref: '20ee04ba1234e9e83eb2ffb5056e23d641c7a018' if Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.2.2')
-end
-group :system_tests do
-  gem "puppet-module-posix-system-r#{minor_version}", require: false, platforms: [:ruby]
-  gem "puppet-module-win-system-r#{minor_version}",   require: false, platforms: [:mswin, :mingw, :x64_mingw]
-  gem "bolt", '~> 1.15',                              require: false if ENV['GEM_BOLT']
+  gem 'puppet_litmus', git: 'https://github.com/puppetlabs/puppet_litmus.git'
```

Comment:
Hello @michaeltlombardi ,
This line can be removed!

Reply:
@lionce it looks like removing this line actually caused things to blow up pretty badly as litmus didn't get installed in appveyor. 🙁 

https://ci.appveyor.com/project/puppetlabs/puppetlabs-service/builds/26668725/job/oxbter4mrpcw5bry

Reply:
I've replaced the line to get the builds to resume for now.

Reply:
@michaeltlombardi  Litmus should be installed when run pdk update. Also ruby version should be >= 2.5.0.

Reply:
PDk updated and looks good now!